content/models.py

```diff
@@ -61,3 +61,24 @@ class EthicalPost(models.Model):
 
 	def __str__(self):
 		return self.title
+
+class CustomPage(models.Model):
+	CHICAGO = "Chicago"
+	DONATIONS = "Donations"
+	FINANCIALS = "Financials"
+	WHYPERU = "WhyPeru"
+	POSTCATEGORY = (
+		(CHICAGO, 'Chicago'),
```

Comment:
I would still have to either use the same image for each header, or make a model for each headers' page. right?